SU2_CFD/src/numerics_direct_mean.cpp

```diff
@@ -4576,11 +4594,18 @@ void CAvgGrad_Base::SetTauJacobian(const su2double *val_Mean_PrimVar,
   const su2double Density = val_Mean_PrimVar[nDim+2];
   const su2double total_viscosity = val_laminar_viscosity + val_eddy_viscosity;
   const su2double xi = total_viscosity/(Density*val_dist_ij);
+  //const su2double xiBulk = 0.0*total_viscosity/(Density*val_dist_ij);
+  //const su2double xiBulk = 10.0*total_viscosity/(Density*val_dist_ij);
+  //const su2double xiBulk = 20.0*total_viscosity/(Density*val_dist_ij);
+  //const su2double xiBulk = 100.0*total_viscosity/(Density*val_dist_ij);
+  //const su2double xiBulk = 1.0*total_viscosity/(Density*val_dist_ij);
+  const su2double xiBulk = 0.0;
```

Comment:
I'm guessing these were corrections to the bulk viscosity.  Is that correct? What's our current understanding here?

Reply:
Yes, that is correct.  I do not have a good understanding.  We should leave out all the bulk viscosity foo now as it is still experimental.

Reply:
Okay.  I'm tempted to make it a cfg option, but I'm unsure of what other options we would include besides a simple "Stokes hypothesis" option.  It doesn't seem like many compressible DNS or compressible LES researchers use anything different.